### ISSUE TITLE
Obtaining very different  alignment curves with the same text input between succesive inference calls

### ISSUE THREAD

**Issue by fatihkiralioglu** [NVIDIA/tacotron2#371]
Hi,
I have trained a custom model and synthesis quality is quite good. However, for some text inputs, alignment can not be achieved in some cases. When I try successive inference calls, I get different aligment graphs for the same text input like the following:
A succesful example:
![image](https://user-images.githubusercontent.com/38240476/84366890-23ccca80-abdc-11ea-9b8e-c06dee63c627.png)
The case when alignment failed and some words are not vocalized.
![image](https://user-images.githubusercontent.com/38240476/84367243-9c338b80-abdc-11ea-94ca-4840a0cc719b.png)

I know that there is some randomness during mel generation dur to dropout but What I wonder is, do this model need more training by dropping learning rate or batch size?
Thanks.

**Comment by neoragex2002** [comment]
seems like some prosody problems. maybe you need prosody annotations in your inputs, depend on what language you are training (i.e. madarin chinese).

**Comment by rafaelvalle** [comment]
It depends on your loss curves. If both training and validation loss are going down you can resume training.

**Comment by aidosRepoint** [comment]
> seems like some prosody problems. maybe you need prosody annotations in your inputs, depend on what language you are training (i.e. madarin chinese).

Hi! Can you please tell me if there is any difference in highlighting types?
I can imagine highlighting stressed vowels, for instance, as:
- making them uppercase. i.g.: thIs Is a cAlculator
- putting a special symbol in front of them. i.g.: th+is +is a c+alculator

I wonder if one has an advantage over another? Have you tried to somehow modify datasets in order to achieve a better (OR I would even say "controlled" prosody)?
